public/js/admin-check-in.js

```diff
@@ -0,0 +1,45 @@
+function httpGetAsync(theUrl, callback){
+  var xmlHttp = new XMLHttpRequest();
+  xmlHttp.onreadystatechange = function() { 
+    if (xmlHttp.readyState == 4 && xmlHttp.status == 200)
+      callback(xmlHttp.responseText);
+  }
+  xmlHttp.open("GET", theUrl, true); // true for asynchronous 
+  xmlHttp.send(null);
+}
+
+var app; 
+
+httpGetAsync('/admin-all-registrants', function(result) {
+  app = new Vue({
+    el: '#app',
+    data: {
+      users: JSON.parse(result).map(u => {
```

Comment:
I am just wondering, how come an arrow function is allowed? Is this bundled with the Vue library?